modules/update-process-workflow.adoc

```diff
@@ -36,6 +38,8 @@ Cluster Operators are updated in separate stages called Runlevels, and the CVO e
 When the CVO deployment is applied, the current CVO pod terminates, and a CVO pod using the new version starts.
 The new CVO proceeds to apply the remaining manifests.
 
+// to do: potentially replace some instances of "apply" in this doc with something like "reconcile" to imply that a lot of these processes are constantly repeating, rather than happening only once.
```

Comment:
There aren't to many `apply` at the moment:

```console
$ w3m -dump -cols 200 https://64077--docspreview.netlify.app/openshift-enterprise/latest/updating/understanding_updates/how-updates-work | grep -i apply
      □ Using RBAC to define and apply permissions
      □ Applying autoscaling to a cluster
 2. The CVO begins applying the manifests from the release image. Cluster Operators are updated in separate stages called Runlevels, and the CVO ensures that all Operators in a Runlevel finish
    to apply the remaining manifests.
Some actions can take significant time to finish. The CVO waits for the actions to complete in order to ensure the subsequent Runlevels can proceed safely. The process of applying all manifests is
declaring the new version as reconciled in its status.versions. The CVO waits until the ClusterOperator reports an acceptable status, and then it will start applying manifests at Runlevel 25.
```

The first two there are table-of-contents entries from other sections.  I'd recommend changing:

>  The CVO begins applying the manifests from the release image.

to:

> The CVO begins reconciling the manifests from the release image.

Changing:

> The new CVO proceeds to apply the remaining manifests.

to:

> The new CVO proceeds to reconcile the remaining manifests.

Changing:

> The process of applying all manifests is expected to take 60 to 120 minutes in total...

to:

> Initially reconciling the new release's manifests is expected to take 60 to 120 minutes in total...

Changing:

> The CVO waits until the ClusterOperator reports an acceptable status, and then it will start applying manifests at Runlevel 25.

to:

> The CVO waits until the ClusterOperator reports an acceptable status, and then it will start reconciling manifests at Runlevel 25.

Push-back welcome, "apply" vs. "reconcile" is a nit-level issue for me, and I'm fine leaving it unaddressed.  I'm also fine going through a few rounds of rewording before finding something everyone likes, or before giving up on finding something everyone likes and leaving it as it stands ;)

Reply:
I don't have a strong opinion on this one. I think this could synergize with the "Maybe I'd really add a "Cluster Version Operator" section" comment above? If we explain better that CVO works continuously it can help conveying the idea of "reconciling" as a long-term process of neverending convergence towards the desired state...